.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Suggest deleting this. Instead, I recommend we add `.vscode` to gitignore and not check in IDE/editor settings